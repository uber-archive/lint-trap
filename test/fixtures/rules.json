[
    {
        "type": "file",
        "file": "/home/uber/code/lint-trap/test/fixtures/rules/globals-invalid.js",
        "errors": [
            {
                "type": "error",
                "linter": "eslint",
                "file": "/home/uber/code/lint-trap/test/fixtures/rules/globals-invalid.js",
                "line": 9,
                "column": 8,
                "rule": "no-undef",
                "message": "'process' is not defined."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "/home/uber/code/lint-trap/test/fixtures/rules/globals-invalid.js",
                "line": 10,
                "column": 8,
                "rule": "no-undef",
                "message": "'global' is not defined."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "/home/uber/code/lint-trap/test/fixtures/rules/globals-invalid.js",
                "line": 11,
                "column": 8,
                "rule": "no-undef",
                "message": "'exports' is not defined."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "/home/uber/code/lint-trap/test/fixtures/rules/globals-invalid.js",
                "line": 12,
                "column": 8,
                "rule": "no-undef",
                "message": "'console' is not defined."
            }
        ]
    },
    {
        "type": "file",
        "file": "/home/uber/code/lint-trap/test/fixtures/rules/globals-valid.js",
        "errors": []
    }
]
