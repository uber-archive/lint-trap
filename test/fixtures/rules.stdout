globals-invalid.js
    error  eslint     9:8    'process' is not defined.
    error  eslint    10:8    'global' is not defined.
    error  eslint    11:8    'exports' is not defined.
    error  eslint    12:8    'console' is not defined.

